### ISSUE TITLE
The intuitive user interface with keyboard navigation

### ISSUE THREAD

**Issue by NiKola-UE** [n42n/n3n#86]
Hello, 

I almost stopped using this account and so I rarely answer here. 

It turns out that my main problem with N3N on the Windows is not really with antiviruses, but it is precisely the lack of a user interface that would fit the keyboard. 

As a blind user who uses the keyboard and screen readers, I cannot work with the mouse, and I do not know in a command line. You might be able to create files in which a wizard or something like users intuitively lead through the whole installation and configuration process, but that should look so that all aspects covered with text so that the screen readers can interpret it well. 

Of course I do not expect to N3N happen soon because this is, it seems to me, still at an early stage of development and is probably not suitable for us without the necessary technical knowledge, but I am sure that the time will come for that; If you understand what I mean exactly.

**Comment by hamishcoleman** [comment]
Duplicates #87
